_Crescent/Entities/Reagents/narcotics.yml

```diff
@@ -146,20 +172,21 @@
   color: "#FF7F7F"
   metabolisms:
     Medicine:
+      metabolismRate: 1 # makes the stim last half as long
       effects:
       - !type:HealthChange
         damage:
           groups:
-            Brute: -1
-          types:
-            Poison: -0.5
-            Heat: -0.15
-            Shock: -0.15
-            Cold: -0.15
-            Radiation: -0.15
+            Brute: -12 # heals -6 per half a second due to faster metabolism
```

Comment:
nice politician speak. im willing to let you off with max -8 at the same rate.

Reply:
the way the guidebook writes it messed me up a bit, its 12 per UNIT instead of 12 per HALF a unit if it had default metabolism.